rolling-update-kubernetes/src/test/scala/akka/rollingupdate/kubernetes/AppVersionRevisionSpec.scala

```diff
@@ -45,188 +23,36 @@ object AppVersionRevisionSpec {
       akka.coordinated-shutdown.terminate-actor-system = off
       akka.coordinated-shutdown.run-by-actor-system-terminate = off
       akka.test.filter-leeway = 10s
+      akka.rollingupdate.kubernetes {
+       api-ca-path = ""
+       api-token-path = ""
+       api-service-host = ""
+       api-service-port = 0
+       namespace = ""
+       namespace-path = ""
+       pod-name = ""
+       secure-api-server = false
+       api-service-request-timeout = 2s
+      }
     """)
 }
-
 class AppVersionRevisionSpec
     extends TestKit(
       ActorSystem(
         "AppVersionRevisionSpec",
-        AppVersionRevisionSpec.config
+        KubernetesApiSpec.config
```

Comment:
Fixed. 

It turns out that tests were passing even without using the right config. Which makes sense because podname defaults to empty in the `reference.conf`. 

Anyway, I keep the new config just to make it very explicit, but did some clean-up as well.